conf/nutch-default.xml

```diff
@@ -1208,6 +1208,28 @@
 </property>
 
 <property>
+  <name>htmlparsefilter.naivebayes.trainfile</name>
+  <value></value>
+  <description>Set the name of the file to be used for Naive Bayes training. The format will be: 
+Each line contains two tab seperted parts
+There are two columns/parts:
+1. "1" or "0", "1" for relevent and "0" for irrelevent document.
+3. Text (text that will be used for training)
+
+Each row will be considered a new "document" for the classifier.
+CAUTION: Set the parser.timeout to -1 or a bigger value than 30, when using this classifier.
+
+  </description>
+</property>
+
+<property>
+  <name>htmlparsefilter.naivebayes.wordlist</name>
+  <value></value>
+  <description>Put the name of the file you want to be used as a list of important words to be matched in the url for the model filter. The format should be one word per line.
```

Comment:
can you insert some line breaks at like 80 chars so it doesn't run off the screen on this? Thanks @asitang